worldInjection.ts

```diff
@@ -1,23 +1,22 @@
-
-import { World } from '@etherealengine/engine/src/ecs/classes/World'
 import { deserializeMap, SCENE_COMPONENT_MAP, SCENE_COMPONENT_MAP_DEFAULT_VALUES, serializeMap, updateMap } from './engine/MapFunctions'
 import { defaultSpatialComponents } from '@etherealengine/engine/src/scene/functions/registerPrefabs'
 import { isNode } from '@etherealengine/engine/src/common/functions/getEnvironment'
+import { Engine } from '@etherealengine/engine/src/ecs/classes/Engine'
 
 export const map = 'Geo Map' as const
 
-export default async (world: World) => {
+export default async () => {
 
   if(!isNode) {
-    (await import('./editor/index')).default(world)
+    (await import('./editor/index')).default()
   }
 
-  world.scenePrefabRegistry.set(map, [
+  Engine.instance.scenePrefabRegistry.set(map, [
     ...defaultSpatialComponents,
     { name: SCENE_COMPONENT_MAP, props: SCENE_COMPONENT_MAP_DEFAULT_VALUES }
   ])
 
-  world.sceneLoadingRegistry.set(SCENE_COMPONENT_MAP, {
+  Engine.instance.sceneLoadingRegistry.set(SCENE_COMPONENT_MAP, {
```

Comment:
sceneLoadingRegistry has been replaced with the `jsonID` on component defintions

respective deserialize, serialize and update functions should be done with component onSet, toJSON and reactor properties